multiversx_sdk_cli/dependencies/modules.py

```diff
@@ -241,7 +241,7 @@ def install(self, overwrite: bool) -> None:
         self._install_sc_meta()
         self._install_wasm_opt()
         self._install_twiggy()
-        self._install_sc_meta_deps()
+        show_message("Please also install all the necessary dependencies required for `sc-meta` to run properly bu running the following command: `sc-meta install all`.")
```

Comment:
Should have been a separate PR from the one adjusting the `amount` for unjailing (for changelog / release note purposes).